src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditRepresentedOrganisationDetails.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿@using EA.Weee.Core.DataStandards
+@using MvcNavigationHelpers
+@using EA.Weee.Web.Controllers
+@using EA.Weee.Web.RazorHelpers
+@using EA.Prsd.Core.Web.Mvc.RazorHelpers
+@using EA.Weee.Web.Areas.Producer.Controllers
+
+@model EA.Weee.Core.Organisations.RepresentingCompanyDetailsViewModel
+@{
+    const string title = "Producer details";
+    ViewBag.Title = title + " - GOV.UK";
+}
+@section full_main_content
+{
+    @this.WeeeGds().BackLink(@Url.UrlFor<OrganisationRegistrationController>(a => a.RepresentingCompanyRedirect()))
+
+    <header class="hgroup">
+        <h1 class="govuk-heading-l">@title</h1>
+    </header>
+
+    <p>Provide the address of the non-UK established Small EEE producer you are registering on behalf of.</p>
+    <p>If the producer has previously registered in the UK under the WEEE Regulations through being a member of a Producer Compliance Scheme, include their Producer Registration Number.</p>
+
+    @using (Html.BeginForm())
+    {
+        @Html.AntiForgeryToken()
+        @Html.Gds().ValidationSummary()
+
+        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.CompanyName)">
```

Comment:
As noticed this should be a readonly field. There is some stuff in the edit organisation details view.